daemonize.go

```diff
@@ -152,7 +152,7 @@ func Run(
 	path string,
 	args []string,
 	env []string,
-	status io.Writer) (err error) {
+	status io.Writer, errLog ...io.Writer) (err error) {
```

Comment:
can you more details to the description around why can't we pass errLog inplace of status.
why is last io.writer list?

Reply:
Added